packages/frontend-shared/src/gql-components/topnav/PromptContent.vue

```diff
@@ -23,6 +23,7 @@
               <img
                 :src="provider.icon"
                 width="14"
+                :alt="provider.name + 'logo'"
```

Comment:
did you want a space here? I think this works?
```suggestion
                :alt="`${provider.name} logo`"
```